commands/lmpop.md

```diff
@@ -0,0 +1,34 @@
+Pops one ore more elements from the first non-empty list key from the list of provided key names.
+
+Similar other commands have limitations:
+- `LPOP` or `RPOP` takes one key, and can return multiple elements.
+- `BLPOP` or `BRPOP` takes multiple keys, but returns one element from just one key.
+
+See `BLMPOP` for the blocking variant of this command.
+
+Search the list from left to right, find the first non-empty list, return the corresponding list key name, pop from the left or right depending on the argument given, and return the elements.
+The number of returned elements will be limited to `count`(default 1) and the list length.
```

Comment:
Elements are popped from either the left or right of the first non-empty list based on the `where` argument. Number of returned elements is limited to the lower between the non-empty list's length and the `count` argument (which defaults to 1).

Reply:
About the `where` arg
Oran said this in the conversation above: I don't think we wanna refer to where IIRC the argument name isn't shown.

Reply:
Elements are popped from either the left or right of the first non-empty list based on the passed argument. The number of returned elements is limited to the lower between the non-empty list's length and the `count` argument (which defaults to 1).